src/components/GamePanel.vue

```diff
@@ -88,27 +91,50 @@ function manualReset() {
     canFlipCards.value = true;
     clearTimeout(resetTimeout);
     allowReset = false;
+    if (firstCard !== undefined && secondCard !== undefined) {
+      firstCard.selection = CardSelection.UNSELECTED;
+      secondCard.selection = CardSelection.UNSELECTED;
+    }
+    firstCard = undefined;
+    secondCard = undefined;
```

Comment:
This makes no sense to me. Change an attributes value before setting it undefined
```suggestion
    firstCard = undefined;
    secondCard = undefined;
```

Reply:
My bad, this should be fine. Just thought that the new assignment of selection will be withdrawn by setting firstCard or secondCard to undefined, but the references of the cards are in the cards reference.